stable_baselines/common/cmd_util.py

```diff
@@ -26,9 +81,11 @@ def make_atari_env(env_id, num_env, seed, wrapper_kwargs=None,
     :param wrapper_kwargs: (dict) the parameters for wrap_deepmind function
     :param start_index: (int) start rank index
     :param allow_early_resets: (bool) allows early reset of the environment
-    :return: (Gym Environment) The atari environment
     :param start_method: (str) method used to start the subprocesses.
         See SubprocVecEnv doc for more information
+    :param use_subprocess: (bool) Whether to use `SubprocVecEnv` or `DummyVecEnv` when
+        `num_env` > 1, `DummyVecEnv` is usually faster. Default: True
```

Comment:
If DummyVecEnv is usually faster (also avoids hassling with processes), why is use of subprocesses default? Matches previous behavior?

Reply:
>Matches previous behavior?

Yes, I'm unsure on that. But in practice, this does not change much for the user, so I this I will edit the default.